services/docker-compose.devel.yml

```diff
@@ -1,54 +1,48 @@
 # Overrides docker-compose.yml config when deploying services in development mode,
 #  i.e. binding with source code in source
-#
 # NOTES:
 # - port 3000 used for ptsv
 #
 version: "3.8"
+x-common-environment: &common-environment
+  DEBUG : "true"
+  SC_BOOT_MODE : debug
+
 services:
   api-server:
     environment:
-      - SC_BOOT_MODE=debug
-      - LOG_LEVEL=DEBUG
-      - DEBUG=true
-      - API_SERVER_PROFILING=${API_SERVER_PROFILING}
+      <<: *common-environment
+      API_SERVER_PROFILING : ${API_SERVER_PROFILING}
     volumes:
       - ./api-server:/devel/services/api-server
       - ../packages:/devel/packages
 
   autoscaling:
     environment:
-      - SC_BOOT_MODE=debug
-      - AUTOSCALING_LOGLEVEL=DEBUG
```

Comment:
so did you replace DEBUG=true instead of loglevel=DEBUG?

Reply:
No, these are two different env vars. 
I will enforce DEBUG in autoscaling in devel-mode. Did not notice i removed this.